### ISSUE TITLE
Containment Cardinality is not respected by the annotator

### ISSUE THREAD

**Issue by umesh-timalsina** [symbench/scm#307]
This will not hinder annotation abilities. But at this point the user can select multiple tags of the same type where the meta rules will prohibit it. Example for a circuit.
![image](https://user-images.githubusercontent.com/11476842/139707120-1645d544-73a4-492b-bf58-5eb92eb06a37.png)
![image](https://user-images.githubusercontent.com/11476842/139707317-c7aa8896-fd5f-4b55-9053-8565cde8946e.png)

**Comment by brollb** [comment]
Yeah, this has crossed my mind before. The hard part is knowing if we want to automatically delete the existing one or change the label somehow or what exactly we would like to do to ensure correctness without disrupting the UX too much.
